src/app/app-platform/ContentAppPlatform.cpp

```diff
@@ -388,14 +389,130 @@ uint32_t ContentAppPlatform::GetPincodeFromContentApp(uint16_t vendorId, uint16_
     return (uint32_t) strtol(pinString.c_str(), &eptr, 10);
 }
 
+constexpr EndpointId kTargetBindingClusterEndpointId = 0;
+constexpr EndpointId kLocalVideoPlayerEndpointId     = 1;
+constexpr EndpointId kLocalSpeakerEndpointId         = 2;
+// the fabric index can't be 0, but needs to be validated as 1
+constexpr FabricIndex kLocalFabricIndex   = 1;
+constexpr ClusterId kNoClusterIdSpecified = kInvalidClusterId;
+
+CHIP_ERROR ContentAppPlatform::ManageClientAccess(OperationalDeviceProxy * targetDeviceProxy, uint16_t targetVendorId,
+                                                  NodeId localNodeId, Controller::WriteResponseSuccessCallback successCb,
+                                                  Controller::WriteResponseFailureCallback failureCb)
+{
+    Access::AccessControl::Entry entry;
+    GetAccessControl().PrepareEntry(entry);
+    entry.SetAuthMode(Access::AuthMode::kCase);
+    entry.SetFabricIndex(kLocalFabricIndex);
+    entry.SetPrivilege(Access::Privilege::kOperate);
+    entry.AddSubject(nullptr, targetDeviceProxy->GetDeviceId());
+
+    std::list<Binding::Structs::TargetStruct::Type> bindings;
+
+    ChipLogProgress(Controller, "Create video player endpoint ACL and binding");
+    {
+        // 0x0006 = OnOff
+        // 0x0503 = Wake On LAN
+        // 0x0504 = Channel
+        // 0x0505 = TargetNavigator
+        // 0x0506 = Media Playback
+        // 0x0507 = Media Input
+        // 0x0508 = Low Power
+        // 0x0509 = Keypad Input
+        // 0x050a = content launcher
+        // 0x050b = Audio Output
+        // 0x050c = Application Launcher
+        // 0x050e = Account Login
+        std::list<ClusterId> allowedClusterList = { 0x0006, 0x0503, 0x0506, 0x0508, 0x0509, 0x050a, 0x050b };
+
+        for (const auto & clusterId : allowedClusterList)
+        {
+            Access::AccessControl::Entry::Target target = { .flags = Access::AccessControl::Entry::Target::kCluster |
+                                                                Access::AccessControl::Entry::Target::kEndpoint,
+                                                            .cluster  = clusterId,
+                                                            .endpoint = kLocalVideoPlayerEndpointId };
+            entry.AddTarget(nullptr, target);
```

Comment:
You'll find that devices may support as few as 3 targets per entry. Example implementation does not have dynamic memory management, so without changing global config flags, or providing a different implementation, this will fail. You can however put several targets into one entry. There's also a limit on the number of entries per fabric which is not currently enforced (but will be), so you can't just make 3 entries for this (and still have entries for admin and other uses). Probably using a device type target is the answer here, but that is not yet supported.

Reply:
updated configuration for TVs to have more entries and more targets